ledger-test-tool/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/UserManagementServiceIT.scala

```diff
@@ -47,6 +49,234 @@ final class UserManagementServiceIT extends UserManagementServiceITBase {
 
   private val AdminUserId = "participant_admin"
 
+  test(
+    "UserManagementUpdateUserIdpWithNonDefaultIdps",
+    "Test reassigning user to a different idp using non default idps",
+    allocate(NoParties),
+  )(implicit ec => { case Participants(Participant(ledger)) =>
+    val userId1 = ledger.nextUserId()
+    val idpId1 = ledger.nextIdentityProviderId()
+    val idpId2 = ledger.nextIdentityProviderId()
+    for {
+      _ <- ledger.createIdentityProviderConfig(identityProviderId = idpId1)
+      _ <- ledger.createIdentityProviderConfig(identityProviderId = idpId2)
+      _ <- ledger.createUser(
+        CreateUserRequest(Some(User(userId1, identityProviderId = idpId1)), Nil)
+      )
+      _ <- ledger.userManagement.updateUserIdentityProviderId(
+        UpdateUserIdentityProviderRequest(
+          userId1,
+          sourceIdentityProviderId = idpId1,
+          targetIdentityProviderId = idpId2,
+        )
+      )
+      get1 <- ledger.userManagement.getUser(
```

Comment:
as in the case of parties, it would make sense to make sure user vanished from the old idp

Reply:
Ok. What we then get is a permission denied